.gitmodules

```diff
@@ -1,21 +1,9 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
-[submodule "lib/openzeppelin-contracts"]
-	path = lib/openzeppelin-contracts
-	url = https://github.com/openzeppelin/openzeppelin-contracts
-[submodule "lib/openzeppelin-contracts-upgradeable"]
-	path = lib/openzeppelin-contracts-upgradeable
-	url = https://github.com/openzeppelin/openzeppelin-contracts-upgradeable
-[submodule "lib/solady"]
-	path = lib/solady
-	url = https://github.com/vectorized/solady
-[submodule "lib/storage-delta"]
-	path = lib/storage-delta
-	url = https://github.com/0xPolygon/storage-delta
-[submodule "lib/deployer-kit"]
-	path = lib/deployer-kit
-	url = https://github.com/0xPolygon/deployer-kit
 [submodule "lib/forge-chronicles"]
```

Comment:
i think we usually default to using solmate dependencies? Any argument against including it here in libs?

Reply:
just keeping it a bit lighter in general, we can always add it when we need it